MotionPhotoReader/app/src/main/java/com/google/android/libraries/motionphotoreader/MotionPhotoReader.java

```diff
@@ -131,7 +208,12 @@ static MotionPhotoReader open(File file,
     }
 
     /**
-     * Sets up and starts a new handler thread for MediaCodec objects (decoder and extractor).
+     * Sets up and starts a new handler thread for the rendering pipeline and media decoders and
+     * extractors.
+     *
+     * A low resolution decoder and extractor are set up for the video track which holds frame data.
+     * If applicable, a high resolution decoder and extractor are set up for the motion track, which
+     * contains video stabilization data.
```

Comment:
what do you mean by high-resolution decoder? We are only decoding the low-res track and not the moments track.